src/bundle-manifest.ts

```diff
@@ -55,9 +55,13 @@ export class Bundle {
   inlinedScripts = new Set<ResolvedUrl>();
   inlinedStyles = new Set<ResolvedUrl>();
 
-  constructor(entrypoints?: Set<ResolvedUrl>, files?: Set<ResolvedUrl>) {
-    this.entrypoints = entrypoints || new Set<ResolvedUrl>();
-    this.files = files || new Set<ResolvedUrl>();
+  constructor(
+      // Filetype discriminator for Bundles.
+      public type: BundleType,
+      // Set of all dependant entrypoint URLs of this bundle.
+      public entrypoints: Set<ResolvedUrl> = new Set(),
```

Comment:
`= new Set<ResolvedUrl>` should work the same?

Reply:
Right! Done.